Assets/Scripts/Models/Need.cs

```diff
@@ -33,18 +30,21 @@ private Need(Need other)
         Type = other.Type;
         LocalisationID = other.LocalisationID;
         Name = other.Name;
-        growthRate = other.growthRate;
+        GrowthRate = other.GrowthRate;
         highToLow = other.highToLow;
         RestoreNeedFurn = other.RestoreNeedFurn;
         RestoreNeedTime = other.RestoreNeedTime;
-        restoreNeedAmount = other.restoreNeedAmount;
-        CompleteOnFail = other.CompleteOnFail;
-        addedInVacuum = other.addedInVacuum;
-        dps = other.dps;
-        luaUpdate = other.luaUpdate;
-        luaOnly = other.luaOnly;
+        RestoreNeedAmount = other.RestoreNeedAmount;
+        Damage = other.Damage;
+
+        if (other.EventActions != null)
```

Comment:
Can we assume that we always have an instance of EventActions?

In Furniture sometimes we assume is there and sometimes we dont... is weird like that.